### PR TITLE
fix(babel): compatibility with create-react-app

### LINKED ISSUE #1364: Should be compatible with create-react-app out of the box
see https://github.com/emmanuelJet/aepp-boilerplate-react/issues/2

### LINKED ISSUE #1413: Bug with argon2-browser
Got such error when was trying to build a base-aepp
`Uncaught TypeError: Cannot destructure property 'ArgonType' of '_aeternity_argon2_browser_dist_argon2_bundled_min_js__WEBPACK_IMPORTED_MODULE_2___default.a' as it is undefined
`

### PR DESCRIPTION
**Pull request by davidyuk**
closes #1364 
depends on https://github.com/aeternity/aepp-calldata-js/pull/114
depends on https://github.com/aeternity/aepp-calldata-js/pull/122
closes #1413 
found issues: https://github.com/dcposch/blakejs/pull/19 https://github.com/swagger-api/swagger-js/issues/2457

### CODE REVIEW

File: babel.config.js [hunk 1]
```diff
@@ -0,0 +1,15 @@
+module.exports = {
+  presets: [
+    ['@babel/preset-env', {
+      include: [
+        // required for create-react-app
+        '@babel/plugin-proposal-nullish-coalescing-operator'
```
Comment (davidyuk) [inline babel.config.js:6]: Would be nice to specify the upper es version instead, https://github.com/babel/babel/issues/6613

File: test/integration/contract-aci.js [hunk 2]
```diff
@@ -520,7 +520,7 @@ describe('Contract instance', function () {
 
       it('Invalid arguments length', async () => {
         await expect(testContract.methods.datTypeGFn())
-          .to.be.rejectedWith('Non matching number of arguments. Got 0 but expected from 1 to 1')
+          .to.be.rejectedWith('Non matching number of arguments. datTypeGFn expects between 1 and 1 number of arguments but got 0')
```
Comment (subhod-i) [inline test/integration/contract-aci.js:523]: Not sure where exactly the error is generated but the error message in here is confusing.

Comment (thepiwo) [reply test/integration/contract-aci.js:523]: agree

**Comment by marc0olo** [comment]
@davidyuk do we need a new release of calldata-lib for this?

File: test/integration/contract-aci.js (hunk 2 above)
Comment (davidyuk) [reply test/integration/contract-aci.js:547]: This was changed on calldata lib side in https://github.com/aeternity/aepp-calldata-js/pull/110
Contract methods may accept a variable amount of arguments if the last arguments are optional. In this case, `datTypeGFn` accepts the one required argument, so we have "between 1 and 1".

**Comment by davidyuk** [comment]
> do we need a new release of calldata-lib for this?

We can merge this as it is now, and switch to the new calldata release before the next SDK release. Without switching to calldata's master, the compatibility issue won't be solved 🤷‍♀️

File: test/integration/contract-aci.js (hunk 2 above)
Comment (subhod-i) [reply test/integration/contract-aci.js:549]: Oh, I see.

**Comment by subhod-i** [comment]
tests are failing. https://github.com/aeternity/aepp-sdk-js/runs/4962960426?check_suite_focus=true#step:8:149

Crypto and Websocekt APIs are tricky. May work for React app but fails for react-native projects.

**Comment by marc0olo** [comment]
@davidyuk new release of calldata-lib is out: https://www.npmjs.com/package/@aeternity/aepp-calldata

**Comment by davidyuk** [comment]
I'm planning to revisit this after merging #1409 #1408 #1407

**Review by thepiwo — APPROVED** [review]

**Pull request merged**